Docs/userguide/quantization_workflow.rst

```diff
@@ -89,70 +39,48 @@ Supported precisions to run inference on target runtimes like |qnn|_ are:
-on-target metrics should be considered depending on your application.
 
-- Latency reduction and/or
-- Memory size reduction
+Quantization workflow
+=====================
 
-If any of the above on-target metrics are not met for your use case, you should consider
-lowering the precision.
+We recommend the following workflow for quantizing a model to improve its efficiency. Unless you have a reason to do otherwise, we recommend you try these actions, listed in ascending order of effort:
 
-The figure below illustrates the recommended quantization workflow and the steps required
-to deploy the quantized model on the target device.
+1. Convert the model to 16-bit floating point (FP16) precision.
+2. Quantize the model to W16A16.
+3. Apply quantization techniques, including post-training quantization (PTQ) and quantization-aware training (QAT).
 
-.. figure:: ../images/quantization_workflow_4.png
+The following sections break these broad actions down into discrete procedures for taking these actions.
 
-   Recommended quantization workflow
+Step 1: Trying FP16 precision (no quantization)
+-----------------------------------------------
 
-FP16 precision (No quantization)
---------------------------------
+We recommend that you start by converting the FP32 model to FP16 precision without quantization. For instructions on how to compile FP16 models for target runtimes, see |qnn_docs|_ or |qai_hub_docs|_.
 
-Converting an FP32 model to FP16 precision without quantization is a recommended starting
-point. For more details on how to compile FP16 models for target runtimes, please refer to
-|qnn_docs|_ or |qai_hub_docs|_.
+If performance is unacceptable at FP16, the next step is to quantize the model.
 
-W16A16 verification
--------------------
+Step 2: Trying W16A16 quantization
+----------------------------------
 
-Before using quantized integer format, it's important to ensure that the FP32 model
-and the quantized model (QuantSim object) perform similarly during the forward pass, especially
-when custom quantizers are included in the model.
+Quantize the model weights and activations to 16-bit integer (W16A16). Do the following:
 
-Set the bit-width to 16 bits for both weights and activations when creating the QuantSim.
-Then, obtain the off-target quantized accuracy metric for the quantized model and verify if
-it aligns with the FP32 model. If it doesn't, please report an issue to |aimet|_.
+1. Ensure that the FP32 model adheres to model-specific guidelines. For instance, in PyTorch QuantSim can only quantize math operations performed by :class:`torch.nn.Module` objects, while :class:`torch.nn.functional` calls will be incorrectly ignored. See framework-specific pages to learn more about such model guidelines.
+2. Once the model conforms to guidelines, create a quantization simulation (QuantSim) version of your model with the bit-width set to 16 bits for both weights and activations (W16A16). See :ref:`<quantsim-workflow>`.
```

Comment:
I don't think there are pages that describe how to to the QuantSim steps the same way there are in the Feature Guide. Can we write standalone pages using that template for:
- Creating a QuantSim model, and
- Testing an off-target model's accuracy?